ScriptEngine/API/LlAPI.h

```diff
@@ -8,13 +8,18 @@ class LlClass {
 public:
     static Local<Value> registerPlugin(const Arguments& args);
     static Local<Value> version(const Arguments& args);
+    static Local<Value> getVersionStatus(const Arguments& args);
+    static Local<Value> getIsDebugMode(const Arguments& args);
+    static Local<Value> getLang(const Arguments& args);
     static Local<Value> versionString(const Arguments& args);
     static Local<Value> requireVersion(const Arguments& args);
-    static Local<Value> listPlugins(const Arguments& args);
+    static Local<Value> listExtPlugins(const Arguments& args);
```

Comment:
Why name it `listExtPlugins`? What does `Ext` mean inside?

Reply:
`Ext` - `extended`. In this method we get extended information about plugins (name, description)

Reply:
sounds not good, i prefer to keep `listPlugins`

Reply:
> sounds not good, i prefer to keep `listPlugins`

okay. and what to do with `listPlugins`, which returns the names of plugins? I suggest calling it `listPluginsName`

Reply:
wait, for compatibility, don't change listPlugins, and replace `listExtPlugins` with `getAllPluginInfo`

Reply:
> wait, for compatibility, don't change listPlugins, and replace `listExtPlugins` with `getAllPluginInfo`

Okay, changed.